security-analysis/security-analysis-api/src/main/java/com/powsybl/security/json/BusResultSerializer.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security.json;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+import com.powsybl.commons.json.JsonUtil;
+import com.powsybl.security.results.BusResult;
+
+import java.io.IOException;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public class BusResultSerializer extends StdSerializer<BusResult>  {
+
+    public BusResultSerializer() {
+        super(BusResult.class);
+    }
+
+    @Override
+    public void serialize(BusResult busResult, JsonGenerator jsonGenerator, SerializerProvider serializerProvider) throws IOException {
+        jsonGenerator.writeStartObject();
+        jsonGenerator.writeStringField("voltageLevelId", busResult.getVoltageLevelId());
```

Comment:
Maybe not to be completely handled in this PR, but shouldn't these results be versionable? Maybe you could only add `jsonGenerator.writeStringField("version", "1.0");` for now and handle it properly if we add a change

Reply:
Bus/branch/3wt results are part of security analysis result which is already versioned.

Reply:
Indeed, you're right... Okay for me then!